types/jscodeshift/index.d.ts

```diff
@@ -1,9 +1,11 @@
 // Type definitions for jscodeshift 0.6
 // Project: https://github.com/facebook/jscodeshift#readme
 // Definitions by: Brie Bunge <https://github.com/brieb>
+//                 Brian Jacobel <https://github.com/bjacobel>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.9
 
+/// <reference path="./src/testUtils.d.ts" />
```

Comment:
I think it'd probably be better to place this in the `tsconfig.json`.

Reply:
I agree, that makes more sense - thank you for the review.

Reply:
Thanks for weighing in -- that's a much better idea! 😄